generators/app/index.js

```diff
@@ -27,7 +27,7 @@ module.exports = yeoman.generators.Base.extend({
   },
 
   install: function () {
-    if (this.options['skip-install']) {
+    if (this.options['skip-install'] || this.options.skipInstall) {
       return;
     }
 
```

Comment:
These dependencies should probably be pushed inside the `bower.json` file. Instead of `bower install --save`ing them